### ISSUE TITLE
Unable to install HLAE

### ISSUE THREAD

**Issue by YusefOuda** [akiver/CSGO-Demos-Manager#283]
from AkiVer/errors.log:

```
2/14/2018 2:04 PM: Access to the path 'C:\Users\Yusef\AppData\Local\AkiVer\hlae\changelog.xml' is denied.
StackTrace:
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at ICSharpCode.SharpZipLib.Zip.FastZip.ExtractFileEntry(ZipEntry entry, String targetName)
   at ICSharpCode.SharpZipLib.Zip.FastZip.ExtractEntry(ZipEntry entry)
   at ICSharpCode.SharpZipLib.Zip.FastZip.ExtractZip(Stream inputStream, String targetDirectory, Overwrite overwrite, ConfirmOverwriteDelegate confirmDelegate, String fileFilter, String directoryFilter, Boolean restoreDateTime, Boolean isStreamOwner)
   at Services.Concrete.Movie.HlaeService.<>c__DisplayClass10_2.<ExtractArchive>b__0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Services.Concrete.Movie.HlaeService.<ExtractArchive>d__10.MoveNext()
-----------------------------------
```

**Comment by akiver** [comment]
Make sure you have the write permission for the folder `C:\Users\Yusef\AppData\Local\AkiVer\`.
Since it's a user folder, you should have it by default.

**Comment by TheKillj0y** [comment]
I'm having the same issue installing HLAE but with different errors in the log. 

-----------------------------------
2/25/2018 10:32 PM: The request was aborted: Could not create SSL/TLS secure channel.
StackTrace:
   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Services.Concrete.Movie.HlaeService.<GetLastReleaseObject>d__8.MoveNext()
-----------------------------------
[errors.log](https://github.com/akiver/CSGO-Demos-Manager/files/1757247/errors.log)

**Comment by akiver** [comment]
@TheKillj0y This issue has been fixed in 2.13.3
